site/src/utils/richParameters.ts

```diff
@@ -4,25 +4,43 @@ import {
 } from "api/typesGenerated";
 import * as Yup from "yup";
 
+export type AutofillSource = "user_history" | "url" | "active_build";
```

Comment:
I haven't reviewed the frontend side thoroughly yet, but I'd rather pull the source information from the backend side, either the previous build or the current one. URL will of course override the auto-fill.

Reply:
Oh, I see. I assumed that it would be mostly a modification of the database query you'd like to introduce.